custom_platforms.js

```diff
@@ -164,21 +600,14 @@ module.exports = {
         "path": "extension/firefox",
         "configuration": {
             "removeRulePatterns": [
-                // Do not exclude scriptlets which contain '$$' when excluding '$$' and '$@$' rules
-                // https://github.com/AdguardTeam/FiltersRegistry/issues/731
-                "^((?!#%#).)*\\$\\$|\\$\\@\\$",
-                "\\$(.*,)?app",
-                "\\$network",
-                "\\$protobuf",
-                "important,protobuf",
-                "\\$extension",
-                ",extension",
-                "\\$hls=",
-                ",hls=",
-                "\\$jsonprune=",
-                ",jsonprune=",
-                "\\$referrerpolicy",
-                ",referrerpolicy"
+                ...HTML_FILTERING_MODIFIER_PATTERNS,
```

Comment:
By the way, do we still need to exclude HTML filtering rules in Firefox extension or perhaps it can be removed now?